src/train/extractor/extractor.py

```diff
@@ -99,6 +116,13 @@ def extract(self, query_results, energy_components, feature_group, energy_source
             feature_power_data = feature_power_data.join(system_feature_data).sort_index().dropna()
         else: 
             feature_power_data = feature_power_data.sort_index()
+        
+        # 4.1 add accelerator features (non aggregated data)
+        if accelerator_flag is not True and len(accelerator_features) > 0:
```

Comment:
Should not add logic here. It is only for the system features which is equal to all container. But, accelerator_feature_data is numerical that should be available only in node-level.